sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -48,7 +48,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.24.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.25.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
nit: we'll want to keep this around so make this an unreleased dependency so CI will pass